docs/howto-openshift-crc.md

```diff
@@ -0,0 +1,267 @@
+running on developers' Linux desktops and requires
+[minimal system resources](https://crc.dev/crc/#minimum-system-requirements-hardware_gsg).
+It also requires virtualization enabled on you local machine.
+
+The instructions in this document where tested with the following
+settings:
+
+```sh
+  $ uname -msr
+  Linux 6.1.18-100.fc36.x86_64 x86_64
+  $ crc version
+  CRC version: 2.15.0+cc05160
+  OpenShift version: 4.12.5
+  Podman version: 4.3.1
+  $ qemu-kvm --version
+  QEMU emulator version 6.2.0 (qemu-6.2.0-17.fc36)
+```
+
+## Setup OpenShift CRC cluster
+Download openshift's crc to your local Linux machine using
+[crc installing instructions](https://crc.dev/crc/#installing_gsg), and
+place the `crc` executable within your `PATH`. Ensure that you have a
+valid installation by [setting up crc](https://crc.dev/crc/#setting-up_gsg):
+
+```sh
+  $ crc version
+  $ crc config view
+  $ crc setup
+```
+
+Make sure that you have an updated pull-secret stored within a local
+file (`pull-secret.txt`), and then start a new crc instance with the
+following command (may take few minutes):
+
+```sh
+  $ crc start -p ./pull-secret.txt
+```
+
+Upon successful deployment, you should see information on how to access
+your cluster, similar to the following example:
+
+```sh
+Started the OpenShift cluster.
+
+The server is accessible via web console at:
+  https://console-openshift-console.apps-crc.testing
+
+Log in as administrator:
+  Username: kubeadmin
+  Password: Y7Dgu-IpHcX-N48UJ-ztphn
+
+Log in as user:
+  Username: developer
+  Password: developer
+
+Use the 'oc' command line interface:
+  $ eval $(crc oc-env)
+  $ oc login -u developer https://api.crc.testing:6443
+
+```
+
+Use the `oc` command line utility to ensure cluster's pods are alive
+and running:
+
+```sh
+  $ eval $(crc oc-env)
+  $ export KUBECTL_CMD=oc
+  $ $KUBECTL_CMD get pods -A
+```
+
+Note that some pods (e.g., `redhat-operators` and `redhat-marketplace`)
+may be in `ImagePullBackOff` status, which is fine in the context of
+this howto document.
+
+When done with the cluster, you may terminate its resources with:
+
+```sh
+  $ crc stop
+  ...
+  $ crc delete
+  ...
+```
+
+## Setup OpenShift samba-SCC
+Samba operator uses a custom
+[security-context-constraints](https://docs.openshift.com/container-platform/4.12/authentication/managing-security-context-constraints.html)
+(SCC) for its pods and containers. Before deploying the samba operator,
+the user should setup the `samba` SCC on the cluster. In order to
+deploy samba SCC manually, execute the following commands:
+
+```sh
+  $ cd samba-operator-dir
+  $ export KUBECTL_CMD=oc
+  $ $KUBECTL_CMD create -f config/openshift/scc.yaml
+  securitycontextconstraints.security.openshift.io/samba created
+  $KUBECTL_CMD get scc/samba -o yaml
+  ...
+```
+
+## Deploy samba-operator
+Enable developer mode and deploy the samba-operator using top-level
+Makefile target `make-deploy`. Wait for `samba-operator` pod to be in
```

Comment:
```suggestion
Makefile target `make-deploy`. Wait for the `samba-operator` pod to be in
```

Reply:
fixed

Reply:
@phlogistonjohn I use vim's spell-checker when editing such docs. Did you use other tools (other then your own eyes) to spot all those issues?

Reply:
Just my :eyes: 
Also, I only speak English so I might notice things that don't "sound right". ;-)